third_party/move/move-compiler-v2/tests/checking/typing/v1-operators/casting_operators_types_mismatch.exp

```diff
@@ -1,48 +1,48 @@
 
 Diagnostics:
-error: operand of cast must be a number
+error: expected `u8|u16|u32|u64|u128|u256|num` but found a value of type `S`
```

Comment:
nit: looks like we use `integer` in some error message contexts (eg, in `binary_gt_invalid.exp`), and `u8|....|num` here. Maybe we could use `integer` here too?

Reply:
Good catch. Done